lang/en/docs/prune-offline-mirror.md

```diff
@@ -0,0 +1,22 @@
+---
+id: docs_prune_offline_mirror
+guide: docs_offline_mirror
+layout: guide
+---
+
+After you configure your offline mirror, Yarn will automatically add new
+package tarballs to the mirror. However, it does not automatically remove
+tarballs that are no longer referenced in `yarn.lock`. For example, if you `$
+yarn remove` a dependency, the tarball will remain in the mirror, even if no
+other dependencies have it has a sub-dependency. This behavior can be desirable
+in a setting where many projects share the same mirror, but when that is not
+the case, you may want to have Yarn remove unnecessary tarballs.
+
+To turn on automating pruning, set `yarn-offline-mirror-pruning` to `true` in
+your `.yarnrc`:
+
+`$ yarn config set yarn-offline-mirror-pruning true`
+
```

Comment:
Currently we don't version the website, over time we will probably do it, the same way we do for React Native.

In our team we check in Yarn js bundle where it is important for all the team members to be on the same version.

I don't think it is a big deal for this feature as anyone changing dependencies after the team memebers with old Yarn would clean everything up.